src/components/button/code/templates/buttonTemplate.jsx

```diff
@@ -3,18 +3,17 @@ var classNames = require('classnames');
 
 module.exports = function(props) {
   var classes = {
-    'btn': true,
-    'btn-primary': props.primary,
-    'btn-secondary': props.secondary,
-    'btn-success': props.success,
-    'btn-info': props.info,
-    'btn-warning': props.warning,
-    'btn-danger': props.danger,
-    'btn-block': props.block,
-    'small': props.small,
-    'large': props.large,
-    'extra-large': props.extraLarge
+    'component-button': true,   
```

Comment:
classnames module allows you to use an array, could be a cleaner option.

```
var classes = ['component-button'];
```

then below..

```
classes.push( props.size );
```